### DISCUSSION TITLE
Community Collaborations

### DISCUSSION THREAD

**Discussion by 4-Leafs-Code** [codinasion-archive/.github-re#48] [General]
Hi Friend! I am the founder and co-owner of [FearlessTech](https://github.com/FearlessTech). I really, very much, like your "Join Us" button and the way you have it submit an issue for people to join your org. I was wondering if you wouldn't mind if I set this up for FearlessTech and telling me how to set up the automation. 

Also, I love Open Source Communities and I wanted to add a redirect to your org from the FearlessTect ReadMe. I've already sent a few members your way including one of my partners. I wanted to ask for your approval first.

**Comment by harshraj8843** [comment]
Hi @4-Leafs-Code!

I'm thrilled that you appreciate our "Join Us" button and are interested in setting up something similar for FearlessTech.

To set up the automatic invitation process, you can use a pre-existing GitHub Action called `invite-on-label`, which you can find at this link: [`https://github.com/vj-abigo/invite-on-label`](https://github.com/vj-abigo/invite-on-label). This action will automatically invite users to join your organization once they submit an issue with a particular label, which can be added by clicking the "Join Us" button.

For reference, you can see how the `EddieHubCommunity` uses this action https://github.com/EddieHubCommunity/support/blob/main/.github/workflows/invitation.yml

Additionally, if you have a specific requirement like adding new members to a particular team on joining, you can create a custom GitHub action to handle that ( like we did 😃 )

It's great to see collaboration between different communities and I'm happy to see that you're promoting our organization by adding a redirect in your FearlessTech README. As long as it's clear that we are separate organizations, we appreciate any support and collaboration with other communities 🤗

**Reply by 4-Leafs-Code** [reply]
Cool Beans, I'll let you know after I've updated it so you can look it over. Thank you so much!!!

**Reply by harshraj8843** [reply]
You're welcome! I'll be happy to review your update whenever you're ready. Just let me know if you have any other questions or if there's anything else I can help you with.

**Reply by 4-Leafs-Code** [reply]
https://github.com/FearlessTech

There, I think I have everything setup and I added a redirect to you at the bottom. Check it out and if you want me to change something, just let me know.

**Reply by harshraj8843** [reply]
It looks so cool 🤩 

But I noticed a bug with the issue template. I am raising a PR for the fix

![Screenshot from 2023-02-18 01-32-34](https://user-images.githubusercontent.com/54644599/219781346-3bec78e3-a1af-4623-ba62-383830fdef67.png)

**Reply by harshraj8843** [reply]
And please don't forget to generate and add `INVITE_TOKEN`

**Reply by 4-Leafs-Code** [reply]
Oh yes I made an invite token. Thank you for fixing this. I had to add the plus sign in the labels section. I fixed that and it was still not triggering the actions. Thank you for fixing this.

**Reply by harshraj8843** [reply]
a simple fix can be, to use a different label. For example `invitation`

**Reply by 4-Leafs-Code** [reply]
I was thinking of using your welcome label. Let me switch it around. It might be the invitation.yml. i just updated it

**Reply by 4-Leafs-Code** [reply]
<img width="949" alt="Screenshot 2023-02-17 171843" src="https://user-images.githubusercontent.com/105990331/219804840-ec56a56f-f66d-42d6-8112-6f1e11692bba.png">

I fixed it. I don't know what the typo was in the yml file but i edited it and saved again. Give it a test run. 😄

**Reply by 4-Leafs-Code** [reply]
Thank you so so much, Friend!!!

**Reply by harshraj8843** [reply]
Glad to help 😄 

Let us know if you want do any project collaboration 🤓

**Reply by 4-Leafs-Code** [reply]
I would love that in the future. Right now I am not able to do much. The facility has had me on a temporary laptop for the last month and I've made very little progress on the reentry repo because of it. But once I get that project done and launched, I'll need some new projects for my org to work on.

**Reply by harshraj8843** [reply]
Sure 🤗

**Comment by everest-dev0323** [comment]
Hi @everyone. I am really happy in a member of this community. I enjoy issue discussion and especially enjoy to make friends. 
Please feel free to contact me and let's discuss further. Thx
